src/components/home/AboutJM.jsx

```diff
@@ -1,7 +1,43 @@
 import React from "react";
+import Image from "next/image";
 
 const AboutJM = () => {
-  return <div>AboutJM</div>;
+  return (
+    <div className="flex">
+      <Image
+        src="/svg/AboutDecor.svg"
+        alt="Placeholder"
+        width={401}
+        height={401}
```

Comment:
Get rid of the width and height attributes, and use tailwind fractional classes instead